Makefile

```diff
@@ -1,22 +1,52 @@
+RHEL5=$(shell grep -q -i 'release 5' /etc/redhat-release && echo 1)
+F27=$(shell grep -q -i 'release 27' /etc/redhat-release && echo 1)
 USE_SELINUX=$(shell test -d /sys/fs/selinux && echo ":Z")
 DOCKERFILE  ?= $(CURDIR)/images/Dockerfile.el7
+ifeq ($(F27), 1) # no pip for F27, only pip3
```

Comment:
Maybe we could do less os-specific checks in this file and check for existence of python3 and pip3 and use those - otherwise fall back to python2 & pip